tests/src/test/java/io/streamnative/pulsar/handlers/kop/BasicEndToEndTestBase.java

```diff
@@ -311,7 +311,7 @@ protected void kafkaConsumeCommitMessage(KConsumer kConsumer,
             ConsumerRecords<Integer, String> records = kConsumer.getConsumer().poll(Duration.ofSeconds(1));
             for (ConsumerRecord<Integer, String> record : records) {
                 Integer key = record.key();
-                assertEquals(messageStrPrefix + key.toString(), record.value());
+                assertEquals(record.value(), messageStrPrefix + key.toString());
```

Comment:
It looks like there is no need to swap these arguments here.

Reply:
TestNG‘s `assertEquals`  method define is `assertEquals(String actual, String expected)`, is different from Junit, we should swap the arguments.

Reply:
Would it make any difference? I mean, if the two overloads have any difference, we must be careful to use `assertEquals` in tests.

Reply:
Yes, the failed message is different.

Example:
```java
@Test
public void test() {
    assertEquals("expected message in junit","actual message in junit");
}
```
In Junit the message should be：
```
org.junit.ComparisonFailure: 
Expected :expected message in junit
Actual   :actual message in junit
```
If we don't swap the arguments, the failed message is wrong.

```
java.lang.AssertionError: 
Expected :actual message in junit
Actual   :expected message in junit
```